src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -89,14 +101,17 @@ public async Task<IEnumerable<IAccount>> GetAccountsAsync()
             }
             else
             {
+                var authParameters = new AuthenticationRequestParameters(
+                        ServiceBundle,
+                        UserTokenCacheInternal,
+                        new AcquireTokenCommonParameters(),
+                        requestContext);
+
+                authParameters.Account = new Account(homeAccountId, null, Authority);
```

Comment:
We need to find a better design for this. It's too contrived that "GetAccount(id)" internally creates an Account object that you pass as a hint for the `GetAccounts` logic.